### ISSUE TITLE
可以监听到 是点击的哪个  路径点

### ISSUE THREAD

**Issue by JoeABCDEF** [antvis/X6#3323]
### 功能描述

<img width="375" alt="image" src="https://user-images.githubusercontent.com/39638510/222025050-46cbc07f-3e2d-4822-9860-1959edf67ef9.png">

### 需求
1. 为了做微调功能,  正常手动微调 只能改小 背景格子的大小, 而且家了吸附功能, 想要正常微调还得 关掉这些, 目前实现 通过上下左右进行 进行 1px 的微调, 但是想可以微调的颗粒度小一点, 可以对线上某个路径点 可进行微调

### 期望解决方案

希望可以获取到 点击哪个 路径点 然后我做些 处理之类的

**Comment by NewByVector** [comment]
你的路径点用的是 vertices 工具吗？

**Comment by JoeABCDEF** [comment]
> 你的路径点用的是 vertices 工具吗？

@NewByVector  是的, 我这个需求是, 想知道点击的哪个路径点(vertices) 可以 使用监听上下左右键进行微调, 通过触发api 上下左右 +- px, 因为手动还得更改 网格大小 还有不吸附啥的, 太麻烦

**Comment by NewByVector** [comment]
你是怎么监听路径点的点击事件的呢？可以给一个 demo？
